<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>liga_dodaj</class>
 <widget class="QDialog" name="liga_dodaj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>Nazwa drużyny / uczestnika</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_29"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Dodaj ligę</string>
   </property>
   <widget class="QWidget" name="layoutWidget_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>Nazwa:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>Dyscyplina:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>Poziom ligi:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>Sezon:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_43">
         <property name="text">
          <string>Miasto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>Ogranizator</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <widget class="QLineEdit" name="lineEdit_23"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_24"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_25"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_5"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_16">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>242</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>Wprowadź hasło</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_22"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_15">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Harmonogram</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edytuj kolejkę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń kolejkę</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kolejka</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>289</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj uczestnika</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_16">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Uczestnicy</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Uczestnik</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edytuj uczestnika</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń uczestnika</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Domyślne zasady:</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Ilość kolejek</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Ilość meczy na kolejkę</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>Liczebność drużyn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>Ilość podmeczy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>Czas trwania podmeczu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>Ilość punktów do zwycięstwa</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QLineEdit" name="lineEdit_16"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_17"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_18"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_19"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_20"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_21"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_18">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>291</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_23">
    <item>
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string>Dodaj kolejkę</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_11">
      <property name="text">
       <string>Wygeneruj harmonogram automatycznie</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
